src/NuGet.Core/NuGet.ProjectModel/ProjectRestoreMetadata.cs

```diff
@@ -223,7 +230,8 @@ public bool Equals(ProjectRestoreMetadata other)
                    EqualityUtility.EqualsWithNullCheck(RestoreLockProperties, other.RestoreLockProperties) &&
                    EqualityUtility.EqualsWithNullCheck(CentralPackageVersionsEnabled, other.CentralPackageVersionsEnabled) &&
                    EqualityUtility.EqualsWithNullCheck(CentralPackageVersionOverrideDisabled, other.CentralPackageVersionOverrideDisabled) &&
-                   EqualityUtility.EqualsWithNullCheck(CentralPackageTransitivePinningEnabled, other.CentralPackageTransitivePinningEnabled);
+                   EqualityUtility.EqualsWithNullCheck(CentralPackageTransitivePinningEnabled, other.CentralPackageTransitivePinningEnabled) &&
+                   RestoreAuditProperties == other.RestoreAuditProperties;
```

Comment:
I guess this should use the same helper function as everything else?  The helper function seems a bit pointless though. It's longer than a direct comparison 😕

Reply:
Actually, looking further up the file, there are plenty of other properties that compare directly with `==`, so no need to change this.